scripts/loadGroundTruth.py

```diff
@@ -24,10 +24,10 @@ def main():
         '--skip_regions', action='store_true', help='expiration time in hours'
     )
     parser.add_argument(
-        '--rgd-auth-cookie',
-        required=False,
+        '--rgd-api-key',
+        required=True,
         type=str,
-        help='RGD Authentication cookie, e.g.: token=<LONG BASE64 STRING>',
+        help='RGD API key',
```

Comment:
This is probably me being lazy but can we default to 'secretKey; it being the default for local dev.  For both this and the regular modelRun loading.